app/src/main/java/org/schabi/newpipe/util/Localization.java

```diff
@@ -274,8 +276,9 @@ public static String getDurationString(final long duration, final boolean isDura
         } else {
             output = String.format(Locale.US, "%d:%02d", minutes, seconds);
         }
+        final String durationPrefix = showDurationPrefix ? "⏱ " : "";
         final String durationPostfix = isDurationComplete ? "" : "+";
-        return output + durationPostfix;
+        return durationPrefix + output + durationPostfix;
```

Comment:
Shouldn't we make this a string resource, especially for RTL languages?

Reply:
I think it's better to keep it this way, since:
- "⏱" is an icon, not really a string, and most of NewPipe's UI sadly does not move icons around based on RTL/LTR
- The duration is formatted with `Locale.US` anyway
- We would probably need 4 strings then